io_scene_niftools/modules/nif_export/scene/__init__.py

```diff
@@ -50,7 +50,7 @@ def get_version_data():
     NifLog.info(f"Writing NIF version 0x{version:08X}")
 
     # get user version and user version 2 for export
-    user_version = b_scene.user_version if b_scene.user_version else b_scene.USER_VERSION.get(game, 0)
-    user_version_2 = b_scene.user_version_2 if b_scene.user_version_2 else b_scene.USER_VERSION_2.get(game, 0)
+    user_version = b_scene.user_version
```

Comment:
now updated by the GUI game change I assume?

Reply:
if they are I must have missed that in the review, or that was done before

Reply:
Yes, user_version and user_version_2 are automatically set by the game, but that was indeed done a while back (by you, it looks like https://github.com/niftools/blender_niftools_addon/commit/fd43ffb085b07c8dda8e4c51826cc9e4a23cf0dd). The removal of the USER_VERSION and USER_VERSION_2 in this was overdue, as they had actually already been removed with the pyffi update, and caused issue #589 .